lib/index.js

```diff
@@ -186,11 +186,11 @@ const Cache = (strapi) => {
         }
       };
 
-      router.post('/content-manager/explorer/:scope',                 bustAdmin);
-      router.post('/content-manager/explorer/:scope/publish/:id*',    bustAdmin);
-      router.post('/content-manager/explorer/:scope/unpublish/:id*',  bustAdmin);
-      router.put('/content-manager/explorer/:scope/:id*',             bustAdmin);
-      router.delete('/content-manager/explorer/:scope/:id*',          bustAdmin);
+      router.post('/content-manager/collection-types/:scope',                 bustAdmin);
+      router.post('/content-manager/collection-types/:scope/publish/:id*',    bustAdmin);
+      router.post('/content-manager/collection-types/:scope/unpublish/:id*',  bustAdmin);
+      router.put('/content-manager/collection-types/:scope/:id*',             bustAdmin);
+      router.delete('/content-manager/collection-types/:scope/:id*',          bustAdmin);
```

Comment:
@roelbeerens i think we should handle `single-types` endpoints too https://github.com/strapi/strapi/blob/387ded4a99a268377f5d88f5503fd03c9630113c/packages/strapi-plugin-content-manager/config/routes.json#L101

Reply:
@cipick Thanks for pointing this out. I'm even considering having the middleware read directly from this file in the future to detect the endpoints.